accelerometer/summariseEpoch.py

```diff
@@ -337,7 +338,7 @@ def perform_wearTime_imputation(e, verbose):
 
 
 
-def calculateECDF(e, inputCol, summary, useRecommendedImputation):
+def calculateECDF(e, inputCol, summary, useRecommendedImputation, intensityDistributionVals = "intensityDistributionVals.txt"):
```

Comment:
Unexpected spaces around keyword / parameter equals (E251)
https://www.flake8rules.com/rules/E251.html
I am not sure if you should prepend `accelerometer/` to the path. Probably yes.